joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/dataflow/DataFlowTests.scala

```diff
@@ -1337,7 +1337,15 @@ class DataFlowTests extends DataFlowCodeToCpgSuite {
       val source = cpg.call("source")
       val sink   = cpg.call("sink")
       val flows  = sink.reachableByFlows(source)
-      flows.map(flowToResultPairs).toSetMutable shouldBe Set(List(("source()", Some(6)), ("sink(source())", Some(6))))
+      flows.map(flowToResultPairs).toSetMutable shouldBe Set(
+        List(
```

Comment:
This flow looks a bit strange. I think we gotta take a closer look here before merging this one. Looks like we're now descending into the sink, coming up from it, and then traversing to the argument `source()`.

Reply:
Actually, it's correct.